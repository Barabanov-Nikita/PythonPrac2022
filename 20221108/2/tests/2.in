(1,1), (1,1), (11,11)
(1,1), (2,2), (11,11)
(1,1), (3,3), (11,11)
qwerwttwf
(1, 2), (3, 4)
1, 2, 3
(1,2), (4,5), (9,9)