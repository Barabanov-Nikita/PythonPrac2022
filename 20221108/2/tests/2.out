Not a triangle
Not a triangle
Not a triangle
Invalid Input
Invalid Input
Invalid Input
1.50
